blockchain/network.py

```diff
@@ -556,12 +562,39 @@ def insert_verifications(self, verification_records):
                 logger().error(message)
                 continue
 
+    def get_min_block_id(self, vrs):
+        """ return minimum block id of given verification records """
+        min_id = 0
+        if vrs:
+            min_id = vrs[0]['block_id']
+            for i in range(1, len(vrs)):
+                id = vrs[i]['block_id']
+                if id < min_id:
+                    min_id = id
+        return min_id
+
+    def _execute_ssc(self, min_block_id):
+        """ execute subscription smart contract """
+        for sc_key in self.processing_node.scp.ssc.keys():
+            # [origin_id:txn_type:phase]
+            criteria = sc_key.split(":")
+            origin_id = criteria[0]
+            txn_type = criteria[1]
+            phase = criteria[2]
+            vrs = verification_db.get_all(origin_id=origin_id, phase=phase, min_block_id=min_block_id)
+            for v in vrs:
+                txns = transaction_db.get_all(txn_type, v['block_id'])
```

Comment:
Also... This may return transactions not included in the verification record if the TT is the same across several subscriptions. We could include owner/origin in the query (e.g. origin - original phase_1 node_id). We also need pagination here if there are more than 10 transactions.